internal/docs/jakartaee/JavaEESpecAssertions.xml

```diff
@@ -277,14 +277,14 @@
                     <section id = "14" name = "Java API for WebSocket 1.1 (WebSocket) Requirements"/>
                     <section id = "15" name = "Java API for JSON Processing 1.1 (JSON-P) Requirements"/>
                     <section id = "16" name = "Java API for JSON Binding 1.0 (JSON-B) Requirements"/>
-                    <section id = "17" name = "Concurrency Utilities for Java EE 1.0 (Concurrency Utilities) Requirements"/>
-                    <section id = "18" name = "Batch Applications for the Java Platform 1.0 (Batch) Requirements"/>
+                    <section id = "17" name = "Concurrency Utilities for Jakarta EE 1.0 (Concurrency Utilities) Requirements"/>
+                    <section id = "18" name = "Batch Applications for the Jakarta Platform 1.0 (Batch) Requirements"/>
```

Comment:
Jakarta Platform 1.0 here looks wrong, the original Java Platform 1.0 looked correct.

Reply:
It's actually the phrasing that looks wrong. 1.0 should maybe go straight after Concurrency Utility such as

Concurrency Utilities 1.0 for Jakarta EE
Batch Application 1.0 for Jakarta Platform

There was no Concurrency Utility and no Batch in Java EE 1.0 so I left it this way.

Reply:
https://github.com/eclipse-ee4j/jakartaee-tck/pull/557#issuecomment-717754319 I think the internal/docs/jakartaee/JavaEESpecAssertions.xml  needs to be regenerated.  There are other issues as well when we compare the same with latest spec document.

In this case the correct text is "Concurrency 2.0 (Concurrency Utilities) Requirements" as per the spec doc.